tmt/steps/report/display.py

```diff
@@ -1,8 +1,22 @@
+import dataclasses
+
 import tmt
 import tmt.steps
 import tmt.steps.report
 from tmt.steps.execute import TEST_OUTPUT_FILENAME
-from tmt.utils import Path
+from tmt.utils import Path, field
+
+
+@dataclasses.dataclass
+class ReportDisplayData(tmt.steps.report.ReportStepData):
+    display_guest: str = field(
+        default='auto',
+        option='--display-guest',
```

Comment:
Probably a bit late with the `html` change already merged, but: What would you say about `--show-guest` as name of the option (and the config key)? Don't know exactly why, but `--display-guest` sounds a bit long :)

Reply:
Once we have custom user configuration the user will be able to store their preference there. 
Personally I prefer 'display-guest'

Reply:
Ok, let's keep it as it is for now. We can do some more polishing in `1.23` if needed.